content/events/2019/xonotic_lan/contents.lr

```diff
@@ -0,0 +1,84 @@
+title: Xonotic LAN
+---
+body:
+
+# Game Night! - Xonotic LAN
+
+<br />
+
+#### Friday, November 15th
+
+##### EN 2036 (CS Lab, 6 to 8pm)
+
+_No registration required, just come on down!_
+
+<br />
+
+#### What is Xonotic?
+
+Xonotic is a modern yet authentic Quake; it runs on the 'DarkPlaces engine', which was initially based on the Quake 1 engine.
+
+As explained by: https://www.xonotic.org/
+
+_"Xonotic is an addictive, arena-style first person shooter with crisp movement and a wide array of weapons. It combines intuitive mechanics with in-your-face action to elevate your heart rate. Xonotic is and will always be free-to-play. It is available under the copyleft-style GPLv3+ license."_
+
+<br />
+
+#### How do I install it?
+
+## Installing on Labnet (MUN Desktops)
+
+If you want to use the desktops at the university to play Xonotic, there's a quicker way to download the game files.
+
+```bash
+# head to your home dir, and make a directory named 'games'
+cd ~
+mkdir games
+cd games
+
+# copy the xonotic zip to said ~/games folder, and unzip it 
+cp ~csclub/games/xonotic-0.8.2.zip .
+unzip xonotic-0.8.2.zip
+
+# execute the linux startup script for xonotic
+cd Xonotic
+./xonotic-linux64-sdl
+
+# or, startup xonotic and directly connect to the server being used for the LAN
+./xonotic-linux64-sdl +connect cssbin
```

Comment:
cssbin is not a valid hostname even within the cs lab. You need to use the full hostname, or instruct the users to configure their hosts file.

Reply:
![tmux__Users_jack_Repos_bd_bln-ui-support-widget-library](https://user-images.githubusercontent.com/6486417/68788295-adec8a80-061d-11ea-8c13-6310af818152.png)

![tmux__Users_jack_Repos_bd_bln-ui-support-widget-library](https://user-images.githubusercontent.com/6486417/68788429-f1df8f80-061d-11ea-8940-0c91649bebe1.png)

tested that very command, and it did work on the lab PCs, as you can see by the screenshot it does mody the /etc/hosts file if you're logged into the desktops, but i'll denote the entire name so those that follow via their own laptop don't get lost

Reply:
149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

Reply:
149f48ab9908c45b5f55f83e452d2f7ddbd4d78d